integrations/extensions/starter-kits/search-with-milvus/search-with-watsonx-data-milvus.md

```diff
@@ -0,0 +1,109 @@
+<img src="./assets/create-milvus-index-watsonx-ai.png" width="1080" height="574" />
+
+4. Click `New collection` to create a new collection.
+5. Provide a unique collection name and choose files to add to the Milvus collection --> click `Create`.
+6. Once the document uploading process is done, you can start testing it in the prompt lab.
+
+See the following screen recording for more details: [./assets/create-milvus-index-watsonx-ai.mov](./assets/create-milvus-index-watsonx-ai.mov).
+
+**NOTE: `document_name` and `text` are the two main fields created in the Milvus collection schema by default. When searching this Milvus collection using custom code, you need to specify these two fields as output_fields. When configuring the Milvus search integration on watsonx Assistant, you need to configure the Title and Body fields with these two fields.**
+
+### Option 2: Ingest data using custom code
+Here is a sample code to ingest documents into Milvus: [../search-with-custom-service/examples/index-with-milvus.py](../search-with-custom-service/examples/index-with-milvus.py). To run it, 
+1. Install dependencies:
+   ```bash
+   python3 -m pip install pymilvus langchain langchain-milvus langchain-ibm ibm-watsonx-ai
+   ```
+2. Create environment variables for Milvus credentials
+  ```bash
+  export MILVUS_HOST = 'Your Milvus GRPC host'
+  export MILVUS_PORT = 'Your Milvus GRPC port'
+  export MILVUS_USER = 'ibmlhapikey' // your Milvus username
+  export MILVUS_PASSWORD = 'Your watsonx.data API key'
+
+  export MILVUS_COLLECTION_NAME = 'Your Milvus collection name' // It can be anything
+  export WATSONX_AI_APIKEY = 'Your watsonx.ai API key' // watsonx.ai embeddings model is used to create vectors
+  export WATSONX_AI_PROJECT_ID = 'Your watsonx.ai project ID' // watsonx.ai project ID is required to access the models
+  ```
+3. Update the `SOURCE_FILE_NAMES`, `SOURCE_URLS`, and `SOURCE_TITLES` variables at the begining of the script to your file names, urls, and titles respectively.
+4. Run the script
+   ```bash
+   python3 index-with-milvus.py
+   ```
+
+## Step 3: Connect to watsonx Assistant
+There are two ways to connect your Milvus vector database to watsonx Assistant for search and conversational search, either through the built-in Milvus search integration or using the Custom Service search.
+
+### Option 1: Use the built-in Milvus search integration
+Please go to the [watsonx Assistant product docs](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-overview) to find more details about how to set up the built-in Milvus search integration.
+
+### Option 2: Use the Custom Service search
+Custom Service search provides a more advanced and flexible way to connect your Milvus vector database to watsonx Assistant. It requires you to set up a custom server that must be reachable by your watsonx Assistant instance. 
```

Comment:
See above comment about using Orchestrate in product name

Reply:
Let's be more specific about what they could do with this option that they cannot do with Option 1.